ui/pages/confirmations/hooks/useConfirmationNetworkInfo.test.ts

```diff
@@ -35,22 +29,14 @@ describe('useConfirmationNetworkInfo', () => {
         ...mockState,
         metamask: {
           ...mockState.metamask,
-          providerConfig: {
-            chainId: '0x7',
-            type: 'rpc',
-            id: 'testNetworkConfigurationId',
-          },
+
+          selectedNetworkClientId: 'networkClientId',
           networkConfigurations: {
-            ...mockState.metamask.networkConfigurations,
-            testNetworkConfigurationId: {
-              rpcUrl: 'https://testrpc.com',
+            networkClientId: {
+              id: 'networkClientId',
               chainId: '0x7',
+              rpcUrl: 'https://testrpc.com',
               nickname: 'Custom Mainnet RPC',
-              type: 'rpc',
-              id: 'testNetworkConfigurationId',
-              rpcPrefs: {
-                imageUrl: './some_image',
```

Comment:
`imageUrl` is a field I'm not sure about.  I don't see it anywhere in the `NetworkController`, yet there are places in extension expecting it in `providerConfig` and `networkConfigurations`.

Reply:
Previous versions of NetworkController also didn't include this field in the state